.github/workflows/build_publish_openmrs.yml

```diff
@@ -20,8 +21,6 @@ jobs:
       HELM_CHART_PATH: bahmni-emr/package/helm/openmrs
     steps:
       - uses: actions/checkout@v2
-      - name: Download hip module omod
```

Comment:
@mohan-13 - I am assuming that you are deleting the HIP build stuff from this file because BahmniIndiaDistro already builds the docker image, and we don't want to re-build/re-publish the HIP image from this repo. Plz confirm. thanks!